.travis/publish_docs.sh

```diff
@@ -6,9 +6,13 @@ sudo chmod 600 ~/.ssh/pulp-infra
 echo "docs.pulpproject.org,8.43.85.236 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBGXG+8vjSQvnAkq33i0XWgpSrbco3rRqNZr0SfVeiqFI7RN/VznwXMioDDhc+hQtgVhd6TYBOrV07IMcKj+FAzg=" >> /home/travis/.ssh/known_hosts
 chmod 644 /home/travis/.ssh/known_hosts
 
-pip3 install -r doc_requirements.txt
+export TEST='doc'
```

Comment:
I'm a little confused about this.  I would expect this script only to run if the ```docs``` flag is set in the environment by the matrix, in which case why re-export it?

Also, is there a distinction between ```doc``` here and the ```docs``` env variable used elsewhere?

Reply:
Good catch. This was supposed to be 'docs'. I want to simulate the same code path in install.sh and before_script.sh that is taken for testing if the docs can build. Except this time I want to run the publish script so they get published to docs.pulpproject.org.